### PR TITLE
EHU-ATJS-20258 - Add "Planet" Field to the Petition for Space Travel Form

### PR DESCRIPTION
**Pull request by Antodi99**
## Task: EHU-ATJS-20258

- [x] Created feature file covering addition of optional "Planet" field in Petition for Space Travel form
- [x] Defined scenarios for field visibility, valid input submission, edge cases, and empty field submission
- [x] Documented test case coverage including positive and edge cases
- [x] Noted automation limitations regarding invalid input validation

### CODE REVIEW

File: ehu-atjs/EHU-ATJS-20258.md [hunk 1]
```diff
@@ -7,12 +7,84 @@
...
+| Positive Input Testing    | Submit valid planet names (normal cases) and verify acceptance and display on output.        | TC-002, TC-005               |
+| Edge Input Testing       | Submit special planet names with numbers, symbols, single characters, verify acceptance/display.  | TC-003, TC-004               |
+| Empty Value Testing      | Submit form with the **Planet** field empty and verify successful submission.                 | TC-006                       |
+
+## Automation Limitations
+
+Validation testing for invalid characters **(TC-007, TC-008)** could not be automated due to current step definition constraints. The test framework’s tooltip validation step uses generic locators which cannot uniquely identify validation errors for the **Planet** field in different dynamic sections (Address Information, Planned Travel Information, Previous Travel Information). This causes ambiguity errors when multiple fields with the same name exist, preventing automated verification of error messages for non-Latin characters or emojis.
```
Comment (ryasrdp) [inline ehu-atjs/EHU-ATJS-20258.md:88]: image  
image
what are the limitations of the framework?!

**Review by ryasrdp — APPROVED** [review]
